src/features/saksoversikt/utils.tsx

```diff
@@ -220,3 +211,58 @@ function statusForPersonligOppmøte(personligOppmøte: Nullable<PersonligOppmøt
             return VilkårVurderingStatus.Uavklart;
     }
 }
+
+const getSatsStatus = (b: Behandling) => {
+    //vi sjekker på behandlingsstatus fordi at man kan endre på vilkår etter sats-steget, som ikke resetter sats.
+
+    if (b.status === Behandlingsstatus.OPPRETTET || b.status === Behandlingsstatus.VILKÅRSVURDERT_AVSLAG) {
+        return VilkårVurderingStatus.IkkeVurdert;
+    }
+    if (
+        b.behandlingsinformasjon.formue &&
+        b.behandlingsinformasjon.formue.borSøkerMedEPS &&
+        b.behandlingsinformasjon.bosituasjon &&
+        b.behandlingsinformasjon.bosituasjon.ektemakeEllerSamboerUførFlyktning !== null
```

Comment:
```suggestion
        b.behandlingsinformasjon.bosituasjon?.ektemakeEllerSamboerUførFlyktning !== null
```

Reply:
Hvis vi optional chainer, for eksempel `b.bosituasjon?.delerBolig !== null` så vil den bli true. noe som er feil :)